### PR TITLE
Skill points are no longer age dependent.

### PR DESCRIPTION
**Pull request by BurpleBineapple**
Everyone is equally incompetent.

🆑 
rscdel: Removes the skill boost from ages.
/ 🆑

### CODE REVIEW

**Comment by gilvian** [comment]
scuffed

**Comment by Firefox13** [comment]
Wasn't it... 8 on old Hestia?

File: code/modules/species/species.dm [hunk 1]
```diff
@@ -794,11 +794,9 @@ The slots that you can use are found in items_clothing.dm and are the inventory
...
 
 /datum/species/proc/skills_from_age(age)	//Converts an age into a skill point allocation modifier. Can be used to give skill point bonuses/penalities not depending on job.
-	switch(age)
-		if(0 to 22) 	. = 0
-		if(23 to 30) 	. = 3
-		if(31 to 45)	. = 6
-		else			. = 8
+	switch(age) //Vesta edit, because people can't behave. All jobs get a flat 4 skill points.
+		if(0 to 100) 	. = 4
+		else			. = 4
```
Comment (Vizoplez) [inline code/modules/species/species.dm:799]: This should be 8 for save consistency reasons

Comment (BurpleBineapple) [reply code/modules/species/species.dm:799]: We're not sticking to hestia's skills. So no

**Pull request merged**